File: src/StudentDB/studentadd.py
import os

print("Student add v1")

name = input("Name of Student >>> ")
id = input("id of Student >>> ")
gender = input("Gender of Student >>> ")
age = input("Age of Student >>> ")
grade = input("Assessed grade of Student >>> ")

file_path = "./src/StudentDB/StudentManage.java"

data_java = f"studentList.add(new StudentInfo({id}, \"{name.capitalize()}\", \"{gender.capitalize()}\", {age}, {grade}));"

with open(file_path, "r") as f:
    content_java = f.readlines()

content_java.insert(-9, "            " + data_java + "\n")

open(file_path, "w").write("".join(content_java))

File: src/StudentDB/StudentAdd.java
package StudentDB;

import org.w3c.dom.ls.LSOutput;

import java.io.*;
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class StudentAdd {
    static Scanner scan = new Scanner(System.in);
    static BufferedWriter writer;
    static BufferedReader reader;
    public static void main(String[] args) {
        System.out.println("Student Add V1");
    }

    static void addStudent(){
        System.out.print("Name of Student >>> ");
        String name = scan.nextLine();

        System.out.print("id of Student >>> ");
        int id = scan.nextInt();
        scan.nextLine();


        System.out.print("Gender of Student >>> ");
        String gender = scan.nextLine();

        System.out.print("Age of Student >>> ");
        int age = scan.nextInt();
        scan.nextLine();

        System.out.print("Assessed grade of Student >>> ");
        int grade = scan.nextInt();
        scan.nextLine();

        StudentInfo studentInfo = new StudentInfo(id, name, gender, age, grade);
        Main.studentInfoList.add(studentInfo);
    }
}
